ci-operator/step-registry/ipi/install/install/full-control/ipi-install-install-full-control-ref.yaml

```diff
@@ -0,0 +1,37 @@
+ref:
+  as: ipi-install-install-full-control
```

Comment:
How about "ipi-install-install-customized"?

Reply:
I do need some candidates for this name, thanks

Reply:
the `full-control` means: I disabled `set -o errexit`, so we have to handle each error manually and determine if exit or not.

Reply:
Better explaining it in the step documentation.

Reply:
Maybe we can rename the filename to be more clear.

Reply:
I'm not sure if it's a good idea that steps in `ipi-install-install` are dependent on various platforms.
Actually, the first half of `ipi-install-install-full-control` is almost the same as `ipi-install-install`,  dividing by platforms will generate lots of redundant code.

Reply:
@jianlinliu @jianli-wei I'm considering using separate steps/scripts for different platforms since there are too many platform-specified codes [1], and also it's hard to maintain if put all codes in the same place, b/c there will be lots of `if` condition statement. 

My concern about the name `ipi-install-install-aws` is that someone may get confused: what's the difference between `ipi-install-install-aws` and `ipi-install-install`? which one should I use for AWS platform? Any other suggestions/comments on this?  maybe `ipi-install-install-aws-customized`?

[1] https://github.com/openshift/release/blob/45c8ce8efa8fae505c1cb7c88355e3fe0f4ea0c2/ci-operator/step-registry/ipi/install/install/full-control/ipi-install-install-full-control-commands.sh#L148-L297

Reply:
1. When I went through `ci-operator/step-registry/ipi/install/install/` folder, I found there is another sub-folder - `stableinitial`, almost the same code as `ipi-install-install`, if take confusion into consideration, stableinitial` already confused enough.
2. Actually the original idea to introduce the new ref yaml file, is because the existing ipi-install-install ref can not meet our requirement, but we would not modify the existing one to avoid missive affect to the existing jobs. Based on that, we are refining the install ref, so introduce a new specific aws install ref yaml file. 

So for me, `ipi-install-install-aws` sounds reasonabl.

Reply:
OK, updating with `ipi-install-install-aws`